extra/utils/permissions.js

```diff
@@ -0,0 +1,24 @@
+
+const permissions =
```

Comment:
when assigning an object, keep the bracket { next to equal to 
let permission = {
    getUsers: {
        all: [],
        read: []
    }
}